_posts/2021-11-09-utah-sgid-statewide-roads-data-layer-updates-11092021.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 11/09/2021
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2021-11-09 10:30:00
+categories:
+  - Featured
+tags:
+  - Roads
```

Comment:
**nitpick**: tags should be lowercased even though we don;t use them
```suggestion
  - roads
```

Reply:
right on.